roles/system_settings/tasks/dns.yml

```diff
@@ -0,0 +1,54 @@
+- name: Append the DNS servers
+  vmware.vmware_rest.appliance_networking_dns_servers:
```

Comment:
maybe we need to split this task into 3 blocks: DNS servers, DNS domains, and DNS hostname, what do you think?

Reply:
What do you mean by blocks?

Reply:
I think it will be more clear like that: 
```
- name: DNS servers
  when: system_settings_dns_mode is defined or system_settings_dns_servers is defined
  block:
  - name: Append the DNS servers
    ...
    when: system_settings_dns_mode_append
  - name: Set the DNS servers
    ...
    when: system_settings_dns_mode_append 
    ...

- name: DNS domains
  when: system_settings_dns_domains
  block:
  - name: Append DNS domains
    ...
    when: system_settings_dns_mode_append
  - name: Set the DNS domains
    ...
    when: not system_settings_dns_mode_append
    ...
```

Reply:
Yeah, that looks definitely, nicer, thanks!